pyccl/boltzmann.py

```diff
@@ -1,18 +1,5 @@
 import numpy as np
 
-try:
-    import classy
-    HAVE_CLASS = True
-except ImportError:
-    HAVE_CLASS = False
-
-try:
-    import camb
-    import camb.model
-    HAVE_CAMB = True
-except ImportError:
-    HAVE_CAMB = False
-
```

Comment:
`isitgr` has a memory leak (which Mustapha has pinpointed as an issue in their repo) so it needs to be reloaded every time. If this is ever fixed then we'll have to get rid of that here as well.

Reply:
Wait, I don't get it. Why do we have this piece of code here (the `import isitgr` below) if we're calling it afterwards (in line 242) anyway?

Reply:
Speaking of which, I replaced the `ImportError` with `ModuleNotFoundError` in the try-except block, because `ImportError` is generic and could be triggered by many things.